### ISSUE TITLE
AIML Using Does not See Display Names

### ISSUE THREAD

**Issue by LokiClifton** [radegastdev/radegast#54]
After Fixing RLV We have started finding that you have to say the Bots Full System Name for it to Know you're talking about it. It needs to Understand its Own username and Reply to others users Display names So it's not Hello BOBB23242P How are you!

**Comment by stonejohnson** [comment]
I agree and it would be so cool if the AIML system would respond to any of
-the username (BOBB23242P  [ignoring case])
-the display name (Robert Palmer or robert palmer [ignore case])
-the first field before a space in the display name (Robert or robert)
